### PR TITLE
Utopia-771 add retention tab

### PR DESCRIPTION
**Pull request by antsand**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/components/public/PIAFormTabs/Accuracy_Retention/index.tsx [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+      }
+    }
+    piaStateChangeHandler(
+      accuracyCorrectionAndRetentionForm,
+      'accuracyCorrectionAndRetention',
+    );
+  };
+
+  const handlePIOptionChange = (event: ChangeEvent<HTMLInputElement>) => {
+    console.log('change');
```
Comment (binggaofw) [inline src/frontend/src/components/public/PIAFormTabs/Accuracy_Retention/index.tsx:60]: is this function we really need if it only do console.log?

Comment (antsand) [reply src/frontend/src/components/public/PIAFormTabs/Accuracy_Retention/index.tsx:60]: correct. I copied this. Forgot to delete this.

**Comment by binggaofw** [comment]
There is one issue with the Yes and No values sent to the backend. I believe the backend expected YES and NO.


Screen Shot 2023-02-23 at 1 28 39 PM

**Comment by antsand** [comment]
> There is one issue with the Yes and No values sent to the backend. I believe the backend expected YES and NO.
> 
> Screen Shot 2023-02-23 at 1 28 39 PM

good catch. Thank you!

File: src/frontend/src/components/public/PIAFormTabs/Accuracy_Retention/index.tsx [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+    }
+    piaStateChangeHandler(
+      accuracyCorrectionAndRetentionForm,
+      'accuracyCorrectionAndRetention',
+    );
+  };
+
+  return (
+    <>
+      <form className="needs-validation">
```
Comment (kushal-arora-fw) [inline src/frontend/src/components/public/PIAFormTabs/Accuracy_Retention/index.tsx:77]: Can we remove this, please?

File: src/frontend/src/routes/router.tsx [hunk 3]
```diff
@@ -93,8 +94,8 @@ const Router = () => {
 
             <Route path="accuracyCorrectionAndRetention">
               <Route index element={<Navigate relative="path" to="view" />} />
-              <Route path="edit" element={comingSoonBanner} />
-              <Route path="view" element={comingSoonBanner} />
+              <Route path="edit" element={<AccuracyCorrectionAndRetention />} />
+              <Route path="view" element={<AccuracyCorrectionAndRetention />} />
```
Comment (kushal-arora-fw) [inline src/frontend/src/routes/router.tsx:98]: Need to create a separate view component

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**